packages/graphql/lib/src/links/websocket_link/websocket_client.dart

```diff
@@ -40,19 +40,19 @@ enum SocketConnectionState { notConnected, handshake, connecting, connected }
 enum ToggleConnectionState { disconnect, connect }
 
 class SocketClientConfig {
-  const SocketClientConfig({
-    this.serializer = const RequestSerializer(),
-    this.parser = const ResponseParser(),
-    this.autoReconnect = true,
-    this.queryAndMutationTimeout = const Duration(seconds: 10),
-    this.inactivityTimeout = const Duration(seconds: 30),
-    this.delayBetweenReconnectionAttempts = const Duration(seconds: 5),
-    this.initialPayload,
-    this.headers,
-    this.connectFn,
-    this.onConnectionLost,
-    this.toggleConnection,
-  });
+  const SocketClientConfig(
+      {this.serializer = const RequestSerializer(),
+      this.parser = const ResponseParser(),
+      this.autoReconnect = true,
+      this.queryAndMutationTimeout = const Duration(seconds: 10),
+      this.inactivityTimeout = const Duration(seconds: 30),
+      this.delayBetweenReconnectionAttempts = const Duration(seconds: 5),
+      this.initialPayload,
+      this.headers,
+      this.connectFn,
+      this.onConnectionLost,
+      this.toggleConnection,
+      this.pingMessage = const <String, dynamic>{}});
```

Comment:
What is the reason that you need exposes this method?

Reply:
This is to send additional information about the ping message to the socket in the payload.

Reply:
This is easy to understand but why you need it?

Reply:
I am using it to send information about the workspaces I want to stay online for because there are multiple workspaces.

Reply:
Yeah @Rochak69 This would help me too. I also want to send extra payload while pinging but there was no option there for ping payload. Please add this ASAP @vincenzopalazzo.